pkg/sql/crdb_internal.go

```diff
@@ -337,6 +338,59 @@ CREATE TABLE crdb_internal.tables (
 	},
 }
 
+var crdbInternalTablesTableLastStats = virtualSchemaTable{
+	comment: "the latest stats for all tables accessible by current user in current database (KV scan)",
+	schema: `
+CREATE TABLE crdb_internal.table_row_statistics (
+  table_id                   INT         NOT NULL,
+  table_name                 STRING      NOT NULL,
+  estimated_row_count        INT         NOT NULL
+)`,
+	populate: func(ctx context.Context, p *planner, db *sqlbase.ImmutableDatabaseDescriptor, addRow func(...tree.Datum) error) error {
+		// Collect the latests statistics for all tables.
+		query := `` +
+			`SELECT s."tableID", max(s."rowCount")` +
```

Comment:
nit you can just put this query in a 
```
`
multiline
string
`
```

and have it formatted via `sqlfum.pt`

Reply:
Done (used multiline string, query was re-formatted).